apps/parsley/src/context/LogContext/types.ts

```diff
@@ -17,7 +17,7 @@ interface LogMetadata {
   buildID?: string;
   execution?: string;
   fileName?: string;
-  groupID?: string;
+  groupIDForCompleteLogs?: string;
```

Comment:
Is this name change necessary? I think its still a groupID whether or not its a full job log or just a test in a job.